platforms/core-execution/build-cache-example-client/src/main/java/org/gradle/caching/example/BuildCacheClientModule.java

```diff
@@ -353,8 +354,8 @@ public void finished(BuildOperationDescriptor buildOperation, OperationFinishEve
     }
 
     @Provides
-    BuildOperationTimeSupplier createTimeSupplier() {
-        return System::currentTimeMillis;
+    Clock createTimeSupplier() {
+        return Time.clock();
```

Comment:
❕ This signals a non-trivial change. Other clients of the library may need to be updated.